vm/src/stdlib/os.rs

```diff
@@ -1138,6 +1143,24 @@ fn os_urandom(size: usize, vm: &VirtualMachine) -> PyResult<Vec<u8>> {
     }
 }
 
+fn os_isatty(fd: i32, _vm: &VirtualMachine) -> bool {
+    unsafe { libc::isatty(fd) != 0 }
+}
+
+fn os_lseek(
+    fd: libc::c_int,
```

Comment:
Not sure about using the `libc` type as arguments. It is unclear to me what type is `libc::c_int` vs libc::c_long` Maybe we should do an explicit conversion?

Reply:
I think it's ok b/c even if they are the same type on a given platform, this function is just a thin wrapper over `lseek` from libc so it's easiest to use the types as they are defined in the `libc` signature.